src/test/javascript/spec/component/shared/notification/notification-sidebar.component.spec.ts

```diff
@@ -243,6 +244,34 @@ describe('Notification Sidebar Component', () => {
             notification.click();
             expect(notificationSidebarComponent.showSidebar).toBeFalse();
         });
+
+        it('should load notifications if none are present', () => {
+            jest.spyOn(notificationSidebarComponent, 'loadNotifications');
+            notificationSidebarComponent.page = 0;
+            notificationSidebarComponent.toggleSidebar();
+            expect(notificationSidebarComponent.loadNotifications).toHaveBeenCalledOnce();
+        });
+
+        it('should not load notifications if some are present', () => {
```

Comment:
```suggestion
        it('should not load notifications repeatedly', () => {
```
Maybe I misunderstand the description of this test. I would appreciate a name that is more descriptive/easier to understand.